proca/lib/proca/server/mtt_worker.ex

```diff
@@ -315,6 +315,7 @@ defmodule Proca.Server.MTTWorker do
       {:mtt, message_id}, email_to.id
     )
     |> Email.from({supporter_name, supporter.email})
+    |> Email.cc(test_email)
```

Comment:
```elixir
iex> Swoosh.Email.new(to: {"john doe", "john.doe@example.com"}) |> Swoosh.Email.cc(nil) |> Map.take([:to, :cc])
%{cc: [], to: [{"john doe", "john.doe@example.com"}]}
iex> Swoosh.Email.new(to: {"john doe", "john.doe@example.com"}) |> Swoosh.Email.cc("test@example.com") |> Map.take([:to, :cc])
%{cc: [{"", "test@example.com"}], to: [{"john doe", "john.doe@example.com"}]}
```

Reply:
@destag, it should only be there when running in test mode, not for real one, of course

Reply:
You're right, I overlooked that. We can have test_email set but not be in test mode